src/styles/components/common/charCounter.scss

```diff
@@ -5,6 +5,7 @@
     position: absolute;
     right: 1rem;
     bottom: -1rem;
+    margin-bottom: .4rem;
```

Comment:
Is this being used to move the CharCounter? The CharCounter is positioned absolutely, meaning that it is positioned with 'top, right, bottom, left' properties already.